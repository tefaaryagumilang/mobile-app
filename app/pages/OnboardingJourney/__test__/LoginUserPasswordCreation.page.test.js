import React from 'react';
import renderer from 'react-test-renderer';
import LoginAccount from '../LoginUserPasswordCreation.page';
import {Provider} from 'react-redux';
import {createStore} from 'redux';
import {reduxForm} from 'redux-form';

const store = createStore(() => ({}));
const DecoratedForm = reduxForm({form: 'loginWithUsernamePassword'})(LoginAccount);

describe('OnboardingJourney: LoginAccount page', () => {
  xit('renders correctly', () => {
    const tree = renderer.create(<Provider store={store}><DecoratedForm /></Provider>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
