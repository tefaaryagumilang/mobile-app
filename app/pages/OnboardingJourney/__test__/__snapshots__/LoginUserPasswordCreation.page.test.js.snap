// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OnboardingJourney: LoginAccount page renders correctly 1`] = `
<KeyboardAwareScrollView
  contentContainerStyle={
    Array [
      Object {
        "backgroundColor": "#FFFFFF",
        "flexGrow": 1,
        "justifyContent": "space-between",
        "paddingBottom": -10,
      },
      Object {
        "padding": 20,
      },
    ]
  }
  extraHeight={120}
  keyboardShouldPersistTaps="handled"
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#000000",
          "fontFamily": "Roboto",
          "fontSize": 24,
          "fontWeight": "500",
          "paddingBottom": 20,
          "paddingHorizontal": 20,
        }
      }
    >
      Username & kata sandi
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#000000",
            "fontWeight": "300",
            "paddingLeft": 20,
          }
        }
      >
        Mohon buat username & kata sandi baru untuk i-banking anda.
      </Text>
    </View>
    <View
      style={
        Object {
          "flexDirection": "column",
          "marginTop": 20,
          "paddingHorizontal": 19,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "space-between",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": undefined,
                "fontSize": 25,
              },
              Object {
                "color": "#666",
                "marginTop": 10,
                "width": 40,
              },
              Object {
                "fontFamily": "simas",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
            ]
          }
        >
          
        </Text>
        <View
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <View>
            <View
              style={
                Array [
                  Object {
                    "backgroundColor": "#ededed",
                    "marginTop": 10,
                  },
                  Object {
                    "backgroundColor": "transparent",
                    "borderBottomWidth": 1,
                    "borderColor": "#ED1D25",
                  },
                ]
              }
            >
              <TextInput
                editable={true}
                maxLength={20}
                onBlur={[Function]}
                onChangeText={[Function]}
                onFocus={[Function]}
                placeholder="Masukkan Username"
                placeholderTextColor={undefined}
                style={
                  Array [
                    Object {
                      "color": "#000",
                      "fontSize": 16,
                      "fontWeight": "300",
                      "height": 50,
                      "padding": 10,
                    },
                    Object {
                      "paddingHorizontal": 0,
                      "paddingVertical": 5,
                    },
                  ]
                }
                underlineColorAndroid="transparent"
                value=""
              />
            </View>
          </View>
        </View>
      </View>
      <View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "space-between",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 25,
                },
                Object {
                  "color": "#666",
                  "marginTop": 10,
                  "width": 40,
                },
                Object {
                  "fontFamily": "simas",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
          <View
            style={
              Object {
                "flex": 1,
              }
            }
          >
            <View>
              <View
                style={
                  Array [
                    Object {
                      "backgroundColor": "#ededed",
                      "marginTop": 10,
                    },
                    Object {
                      "backgroundColor": "transparent",
                      "borderBottomWidth": 1,
                      "borderColor": "#ED1D25",
                    },
                  ]
                }
              >
                <TextInput
                  editable={true}
                  onBlur={[Function]}
                  onChangeText={[Function]}
                  onFocus={[Function]}
                  placeholder="Masukkan kata sandi"
                  placeholderTextColor={undefined}
                  secureTextEntry={true}
                  style={
                    Array [
                      Object {
                        "color": "#000",
                        "fontSize": 16,
                        "fontWeight": "300",
                        "height": 50,
                        "padding": 10,
                      },
                      Object {
                        "paddingHorizontal": 0,
                        "paddingVertical": 5,
                      },
                    ]
                  }
                  underlineColorAndroid="transparent"
                  value=""
                />
              </View>
            </View>
          </View>
        </View>
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          hitSlop={undefined}
          isTVSelectable={true}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
              "position": "absolute",
              "right": 0,
              "top": 15,
              "width": 30,
            }
          }
          testID={undefined}
          tvParallaxProperties={undefined}
        >
          <Image
            source={1}
          />
        </View>
      </View>
      <View>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#666",
              "fontFamily": "Roboto",
              "fontSize": 12,
              "fontWeight": "400",
              "paddingLeft": 50,
              "paddingTop": 4,
            }
          }
        >
          Minimal 8 karakter. Alfanumerik. Gunakan minimal 1 huruf besar.
        </Text>
        <View>
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "space-between",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={false}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": undefined,
                    "fontSize": 25,
                  },
                  Object {
                    "color": "#666",
                    "marginTop": 10,
                    "width": 40,
                  },
                  Object {
                    "fontFamily": "simas",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                ]
              }
            >
              
            </Text>
            <View
              style={
                Object {
                  "flex": 1,
                }
              }
            >
              <View>
                <View
                  style={
                    Array [
                      Object {
                        "backgroundColor": "#ededed",
                        "marginTop": 10,
                      },
                      Object {
                        "backgroundColor": "transparent",
                        "borderBottomWidth": 1,
                        "borderColor": "#ED1D25",
                      },
                    ]
                  }
                >
                  <TextInput
                    editable={true}
                    onBlur={[Function]}
                    onChangeText={[Function]}
                    onFocus={[Function]}
                    placeholder="Konfirmasi ulang kata sandi"
                    placeholderTextColor={undefined}
                    secureTextEntry={true}
                    style={
                      Array [
                        Object {
                          "color": "#000",
                          "fontSize": 16,
                          "fontWeight": "300",
                          "height": 50,
                          "padding": 10,
                        },
                        Object {
                          "paddingHorizontal": 0,
                          "paddingVertical": 5,
                        },
                      ]
                    }
                    underlineColorAndroid="transparent"
                    value=""
                  />
                </View>
              </View>
            </View>
          </View>
          <View
            accessibilityComponentType={undefined}
            accessibilityLabel={undefined}
            accessibilityTraits={undefined}
            accessible={true}
            hitSlop={undefined}
            isTVSelectable={true}
            onLayout={undefined}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
                "position": "absolute",
                "right": 0,
                "top": 15,
                "width": 30,
              }
            }
            testID={undefined}
            tvParallaxProperties={undefined}
          >
            <Image
              source={1}
            />
          </View>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "paddingVertical": 20,
        }
      }
    />
  </View>
  <View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#ED1D25",
              "borderRadius": 30,
              "flexDirection": "row",
              "height": 47,
              "justifyContent": "center",
              "paddingHorizontal": 15,
            },
            undefined,
            Object {},
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#FFFFFF",
              "fontSize": 16,
            }
          }
        >
          BERIKUTNYA
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "paddingVertical": 10,
        }
      }
    >
      <View>
        <View>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "textAlign": "center",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "#9B9B9B",
                  "fontSize": 12,
                  "textAlign": "center",
                }
              }
            >
               Butuh bantuan? 
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              onPress={[Function]}
              style={
                Object {
                  "color": "#ED1D25",
                  "fontSize": 12,
                  "textAlign": "center",
                }
              }
            >
              Hubungi kami
            </Text>
          </Text>
        </View>
      </View>
    </View>
  </View>
</KeyboardAwareScrollView>
`;
